### PR TITLE
New attempt at adding 1.15.2 burger extraction

### PR DESCRIPTION
**Pull request by uncovery**

### CODE REVIEW

**Comment by rom1504** [comment]
Still no blocks.json. either add it either don't put it in load paths

**Comment by uncovery** [comment]
Tried to add the blocks, they are listed here now, let me know if it worked, thanks!

File: data/dataPaths.json [hunk 1]
```diff
@@ -465,7 +465,11 @@
     "1.15.2": {
       "protocol": "pc/1.15.2",
       "version": "pc/1.15.2",
-      "language": "pc/1.15.2"
+      "language": "pc/1.15.2",
+      "blocks": "pc/1.15.2",
+      "entities": "pc/1.15.2",
+      "items": "pc/1.15.2",
+      "recipies": "pc/1.15.2"
```
Comment (rom1504) [inline data/dataPaths.json:472]: This should be recipes, not recipies

**Comment by uncovery** [comment]
Sorry, this was a bit sloppy, I will try to be more careful for future changes.

**Comment by rom1504** [comment]
It's fine

**Pull request merged**